### PR TITLE
added .grid class and updated various components to use it

### PR DESCRIPTION
**Pull request by Anon247**
Done to support a much more flexible grid layout. Also removes the need to change potentially 100's of classNames in the DOM at the same time.

### CODE REVIEW

**Comment by Anon247** [comment]
Still seem to be struggling with line seperators... I'll hold off on that commit for now

**Comment by InfiniteTF** [comment]
I don't think the scene card change is worthwhile. When rapidly changing the zoom levels the performance is definitely worse with grid than the existing flexbox layout.

The performer page layout is an improvement though, both in terms of performance and simplicity. However, I think there should be a bit more whitespace, and maybe a smaller max-height on the performer image since it takes over the entire page

**Comment by Anon247** [comment]
Fair enough, I'll update the PR to remove the grid scene page stuff

**Comment by Anon247** [comment]
@InfiniteTF Added those changes you requested and rebased onto latest commit so no need to worry about conflicts, please review.

File: ui/v2.5/src/components/Performers/styles.scss [hunk 1]
```diff
@@ -7,19 +7,24 @@
...
+  &.grid{
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto 1fr;
+  }
+
+  .performer-image {
+    grid-row: 1 / 3;
+    cursor: pointer;
+    max-width: 50vw;
+    max-height: 70vh;
```
Comment (WithoutPants) [inline ui/v2.5/src/components/Performers/styles.scss:22]: Why 70vh? Ends up with a lot of wasted vertical space underneath.

**Comment by WithoutPants** [comment]
Closing this for inactivity for now. Feel free to reopen if needed.

**Pull request closed without merging**